cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -1106,6 +1107,13 @@ def build_dispatcher_response(self,
         if query_out is not None:
             out_dict['products'] = query_out.prod_dictionary
             out_dict['exit_status'] = query_out.status_dictionary
+            if getattr(self.instrument, 'unknown_arguments_name_list', []):
+                if len(self.instrument.unknown_arguments_name_list) == 1:
+                    comment = f'Please note that argument {self.instrument.unknown_arguments_name_list[0]} is not used'
+                else:
+                    comment = f'Please note that arguments {", ".join(self.instrument.unknown_arguments_name_list)} are not used'
+                out_dict['exit_status']['comment'] = \
+                    out_dict['exit_status']['comment'] + ' ' + comment if out_dict['exit_status']['comment'] else comment
```

Comment:
do we not have a warning field too?

Reply:
We do. We manually putted the message about unused arguments (e.g. coords for GW) into the comment field, so I used it here also.

Reply:
The comment field is used within the frontend to be visualized with the yellow background. We had discussed here

https://github.com/oda-hub/dispatcher-app/issues/443#issuecomment-1233034425

Reply:
yeah that's what I meant. there is another comment field, behaves like comment but more alarming.

Reply:
We used 'comment' field before to warn about unused parameters, so why change it?

Reply:
it does seem like warning should go into warning field.
but ok, this is more of a comment than a warning. It does not break the result, just means that the request is redundant.